scripts/install_stan.R

```diff
@@ -0,0 +1,6 @@
+# install stan
+# users should run this script to install stan their R session
+
+install.packages("cmdstanr", repos = c("https://mc-stan.org/r-packages/", getOption("repos")))
+library(cmdstanr)
+install_cmdstan()
```

Comment:
Does this file need to be an R script? The other script files are shell scripts.

Reply:
Good point @eitsupi.  I've changed it to a shell script.  Thanks!